workflow/basics/context.mdx

```diff
@@ -79,120 +98,262 @@ export const { POST } = serve<string>(
   },
 );
 ```
+
+```python Serial execution (Python)
+from fastapi import FastAPI
+from upstash_workflow.fastapi import Serve
+from upstash_workflow import AsyncWorkflowContext
+
+app = FastAPI()
+serve = Serve(app)
+
+
+@serve.post("/api/example")
+async def example(context: AsyncWorkflowContext[str]) -> None:
+    input = context.request_payload
+
+    async def _step1():
+        return some_work(input)
+
+    result1 = await context.run("step-1", _step1)
+
+    async def _step2():
+        return some_other_work(result1)
+
+    await context.run("step-2", _step2)
+
+```
+
 </CodeGroup>
 
 ### context.sleep
 
 Pauses workflow execution for a specified duration.
 
-<Tip>
-  Always `await` a `sleep` action to properly pause execution.
-</Tip>
+<Tip>Always `await` a `sleep` action to properly pause execution.</Tip>
 
-```typescript api/workflow/route.ts
-import { serve } from "@upstash/workflow/nextjs"
-import { signIn, sendEmail } from "@/utils/onboarding-utils"
+<CodeGroup>
 
-export const { POST } = serve<User>(
-  async (context) => {
-    const userData = context.requestPayload;
+```typescript JS
```

Comment:
shall we replace JS with TypeScript since we use Python, not Py?

In the other files and docs too, like https://github.com/upstash/docs/pull/365/files#diff-c5267b4819b45c3401268ef7c3a2e19045f915abac24a63563a43d18ba0c579cR34